include/cuco/static_multimap.cuh

```diff
@@ -297,11 +296,9 @@ class static_multimap {
    * @param first Beginning of the sequence of keys
    * @param last End of the sequence of keys
    * @param stream CUDA stream used for insert
-   *
-   * @return Number of successful insertions
    */
   template <typename InputIt>
-  size_type insert(InputIt first, InputIt last, cuda::stream_ref stream = {});
+  void insert(InputIt first, InputIt last, cuda::stream_ref stream = {});
```

Comment:
So we remove the reduction from `insert` but introduce it again in `insert_if`?

Reply:
Yes, the cleanup I was aiming:

- multiset/multimap `insert` returning `void` since all inputs will be inserted so the reduction is redundant and more expensive than needed
- multiset/multimap `insert_if` returning `size_type` in case we want to track the number of conditional insertions
- multiset/multimap `insert_if_async` returning `void` for the cases that the #insertions is not desired

Reply:
Totally makes sense. Thanks for the explanation.